styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -596,9 +599,19 @@ private Entity backfillToEntity(Backfill backfill) {
     return builder.build();
   }
 
-  public Optional<Long> globalConcurrency() {
+  Optional<Long> globalConcurrency() {
     return getOpt(datastore, globalConfigKey)
         .filter(e -> e.contains(PROPERTY_CONFIG_CONCURRENCY))
         .map(e -> e.getLong(PROPERTY_CONFIG_CONCURRENCY));
   }
+
+  List<String> clientBlacklist() {
+    final String blacklist = readConfigString(PROPERTY_CONFIG_CLIENT_BLACKLIST, "");
+    if (blacklist.isEmpty()) {
+      return ImmutableList.of();
+    } else {
+      return Arrays.stream(blacklist.split(";")).map(
```

Comment:
Datastore supports lists of values natively. That's probably better than manual splitting on `;`.